rust/main/agents/relayer/src/msg/metadata/ccip_read.rs

```diff
@@ -56,13 +54,18 @@ impl MetadataBuilder for CcipReadIsmMetadataBuilder {
                 return Err(MetadataBuildError::CouldNotFetch);
             }
             Err(raw_error) => {
-                let matching_regex = Regex::new(r"0x[[:xdigit:]]+")
-                    .map_err(|_| MetadataBuildError::FailedToBuild)?;
+                let matching_regex = Regex::new(r"0x[[:xdigit:]]+").map_err(|_| {
+                    MetadataBuildError::FailedToBuild("Invalid matching regex".into())
+                })?;
```

Comment:
here and elsewhere as well
```suggestion
                let matching_regex = Regex::new(r"0x[[:xdigit:]]+")
                    .map_err(|err| MetadataBuildError::FailedToBuild(err.to_string()))?;
```